packages/react/src/components/headers/HeaderWrapper.tsx

```diff
@@ -6,10 +6,29 @@ import {Content, IContentProps} from '../content/Content';
 import {ITabsHeaderProps, TabsHeader} from './TabsHeader';
 
 export interface IHeaderWrapperProps extends ITabsHeaderProps {
+    /**
+     * Text that appears above the tabs in the header wrapper
+     */
     description?: string | React.ReactNode;
+    /**
+     * Add an action bar/ action button

```

Comment:
I think it would be relevant to have an example of this

```suggestion
     * Action buttons displayed in the right portion of the header
```